plugins/inputs/internet_speed/internet_speed_test.go

```diff
@@ -13,6 +13,8 @@ func TestGathering(t *testing.T) {
 	}
 	internetSpeed := &InternetSpeed{
 		EnableFileDownload: false,
+		Offset:             "2s",
+		Cache:              true,
```

Comment:
Please add a specific test for those features instead of just pasting the settings to an existing test without actually testing if the settings took effect.